src/main/java/com/coveros/selenified/utilities/Reporter.java

```diff
@@ -901,6 +880,165 @@ public static String formatKeyPair(Map<String, Object> keyPairs) {
         return stringBuilder.toString();
     }
 
+    /**
+     * Formats the request parameters to be 'prettily' printed out in HTML
+     *
+     * @param params - the parameters to be formatted. Either a JSON object, or a
+     *               hashmap
+     * @return String: a 'prettily' formatted string that is HTML safe to output
+     */
+    public static String getRequestPayloadOutput(Request params, File file) {
+        StringBuilder payload = new StringBuilder();
+        String uuid = getUUID();
+        if ((params != null && params.isPayload()) || file != null) {
+            payload.append("<a href='javascript:void(0)' onclick='toggle(\"").append(uuid).append("\")'>Toggle Payload</a> ");
+            payload.append("<span id='").append(uuid).append("' style='display:none;'>");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "' style='display:none;'>" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "<span id='" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)